genie/database_to_staging.py

```diff
@@ -35,6 +35,73 @@
 SV_CENTER_PATH = os.path.join(GENIE_RELEASE_DIR, "data_sv_%s.txt")
 BED_DIFFS_SEQASSAY_PATH = os.path.join(GENIE_RELEASE_DIR, "diff_%s.csv")
 
+FULL_MAF_RELEASE_COLUMNS = [
```

Comment:
Are these all the columns in the release maf right now?

Reply:
Yeah, I looked at the [AACR maf file columns from the 16.1 consortium release](https://www.synapse.org/#!Synapse:syn5571527) and used those columns